README.md

```diff
@@ -21,13 +21,34 @@
 - Editor wars
 - Wars ...
 - Superiority of FP
+
 # Setup
+Setup Emacs with https://github.com/koljure/beginning-clojure-emacs.d  
+* General 
   - A terminal client
   - Java 8 or higher
   - [Clojure](https://clojure.org/)
   - [Leiningen](https://leiningen.org/)
-  - [Emacs](https://www.gnu.org/software/emacs/)
-  - Setup Emacs with https://github.com/koljure/beginning-clojure-emacs.d
+  - [Emacs](https://www.gnu.org/software/emacs/) 
+
+* Sample setup guide (for Mac)
+  - [Install JRE on Mac](https://java.com/en/download/help/mac_install.xml)   
+    *or*       
+    use *Brew* package manager `brew cask install java`   
+
+  - Install **Clojure** using *Brew* package manager- `brew install clojure`  
+  - Install **Leiningen** using *Brew* package manager- `brew install leiningen`    
+  - Install **Emacs** using *Brew* package manager- `brew cask install emacs`  
+      Setup Emacs on Mac-  
+      1) `git clone https://github.com/koljure/beginning-clojure-emacs.d.git`
+      2) `cd beginning-clojure-emacs.d/ && cp init.el ~/.emacs.d`  
```

Comment:
@souvikhaldar This file can be copied directly without having to `cd` first

Reply:
Yes. Fixed it. @trycatcher